<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1606</width>
    <height>888</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TMV3 Viewer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="7">
    <widget class="QPushButton" name="BtnExit">
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 170, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="6">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #303030;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <zorder>mpl</zorder>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="Btn21">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Templates/Oxygen-Icons.org-Oxygen-Actions-go-up.ico</normaloff>../Templates/Oxygen-Icons.org-Oxygen-Actions-go-up.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="7" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label21">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label22V">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label22">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QLabel" name="label21V">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>None</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label23">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnV2">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>140</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>240</width>
         <height>140</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label24">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label24V">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label25">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label25V">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label11">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label11V">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label12">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label12V">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label13">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnV1">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>140</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>240</width>
         <height>140</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label14">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label14V">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label15">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label15V">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>130</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="Btn22">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Templates/Oxygen-Icons.org-Oxygen-Actions-go-down.ico</normaloff>../Templates/Oxygen-Icons.org-Oxygen-Actions-go-down.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="BtnAddLimit">
       <property name="text">
        <string>add Limit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnSave">
       <property name="text">
        <string>save image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnPrint">
       <property name="text">
        <string>print</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnPDF">
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" alignment="Qt::AlignBottom">
    <widget class="QPushButton" name="Btn12">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Templates/Oxygen-Icons.org-Oxygen-Actions-go-down.ico</normaloff>../Templates/Oxygen-Icons.org-Oxygen-Actions-go-down.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="8">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignTop">
    <widget class="QPushButton" name="Btn11">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Templates/Oxygen-Icons.org-Oxygen-Actions-go-up.ico</normaloff>../Templates/Oxygen-Icons.org-Oxygen-Actions-go-up.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QFrame" name="mpl">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
